spec/migrations/umm_c_1_15_5_to_1_16_spec.rb

```diff
@@ -0,0 +1,33 @@
+require File.join(Rails.root, 'db', 'migrate', '20210312165348_umm_c_1_15_5_to_1_16')
```

Comment:
I think it's good that you did this, to stay in line with the process. I'm wondering, though, for such a simple migration in the future, if it is worth writing the migration tests for.

Reply:
Yeah, I'm not really sure. Part of the reason the tests are helpful is just to see if the migration actually works (while I'm writing the migration) because it's just more convenient than actually executing the migration.